Analysis/Tasks/PWGHF/HFCandidateCreator2Prong.cxx

```diff
@@ -179,6 +181,228 @@ struct HFCandidateCreator2ProngMC {
+                  full::RSecondaryVertex,
+                  full::DecayLength,
+                  full::DecayLengthXY,
+                  full::DecayLengthNormalised,
+                  full::DecayLengthXYNormalised,
+                  full::ImpactParameterNormalised0,
+                  full::PtProng0,
+                  full::PProng0,
+                  full::ImpactParameterNormalised1,
+                  full::PtProng1,
+                  full::PProng1,
+                  hf_cand::PxProng0,
+                  hf_cand::PyProng0,
+                  hf_cand::PzProng0,
+                  hf_cand::PxProng1,
+                  hf_cand::PyProng1,
+                  hf_cand::PzProng1,
+                  hf_cand::ImpactParameter0,
+                  hf_cand::ImpactParameter1,
+                  hf_cand::ErrorImpactParameter0,
+                  hf_cand::ErrorImpactParameter1,
+                  full::NSigTPC_Pi_0,
+                  full::NSigTPC_Ka_0,
+                  full::NSigTOF_Pi_0,
+                  full::NSigTOF_Ka_0,
+                  full::NSigTPC_Pi_1,
+                  full::NSigTPC_Ka_1,
+                  full::NSigTOF_Pi_1,
+                  full::NSigTOF_Ka_1,
+                  full::M,
+                  full::ImpactParameterProduct,
+                  full::CosThetaStar,
+                  full::Pt,
+                  full::P,
+                  full::CPA,
+                  full::CPAXY,
+                  full::Ct,
+                  full::Eta,
+                  full::Phi,
+                  full::Y,
+                  full::E,
+                  full::MCflag);
+
+DECLARE_SOA_TABLE(HfCandProng2FullEvents, "AOD", "HFCANDP2FullE",
+                  collision::BCId,
+                  collision::NumContrib,
+                  collision::PosX,
+                  collision::PosY,
+                  collision::PosZ,
+                  full::IsEventReject,
+                  full::RunNumber);
+
+DECLARE_SOA_TABLE(HfCandProng2FullParticles, "AOD", "HFCANDP2FullP",
+                  collision::BCId,
+                  full::Pt,
+                  full::Eta,
+                  full::Phi,
+                  full::Y,
+                  full::MCflag);
+
+} // namespace o2::aod
+
+/// Writes a debug tree
+struct CandidateTreeWriter {
+  Produces<o2::aod::HfCandProng2Full> rowCandidateFull;
+  Produces<o2::aod::HfCandProng2FullEvents> rowCandidateFullEvents;
+  Produces<o2::aod::HfCandProng2FullParticles> rowCandidateFullParticles;
+  void init(InitContext const&)
+  {
+  }
+  void process(aod::Collisions const& collisions,
+               aod::McCollisions const& mccollisions,
+               soa::Join<aod::HfCandProng2, aod::HfCandProng2MCRec> const& candidates,
+               soa::Join<aod::McParticles, aod::HfCandProng2MCGen> const& particles,
+               aod::BigTracksPID const& tracks)
+  {
+    int ncol = 0;
+    rowCandidateFullEvents.reserve(collisions.size());
+    Printf("Collisions");
```

Comment:
Are all the `Printf` messages supposed to be kept?
If meant for debugging, I would suggest to comment them out to avoid spamming the log.

Reply:
Thanks for pointing this out @vkucera . In fact Printf should never be used, please use LOGF or LOGP with level DEBUG for things which you don't want to see. Those are compiled away in a cvmfs build

Reply:
Sorry I missed the Printf and the comment, I will remove them!